Mathlib/NumberTheory/NumberField/Discriminant/Basic.lean

```diff
@@ -153,10 +153,12 @@ theorem exists_ne_zero_mem_ringOfIntegers_of_norm_le_mul_sqrt_discr :
   simp_rw [Units.val_one, FractionalIdeal.absNorm_one, Rat.cast_one, one_mul] at h_nm
   exact h_nm
 
-variable {K}
-
-theorem abs_discr_ge (h : 1 < finrank ℚ K) :
-    (4 / 9 : ℝ) * (3 * π / 4) ^ finrank ℚ K ≤ |discr K| := by
+/--
+A stronger version of `abs_discr_ge` but probably less easy to use in practice.
+-/
+theorem abs_discr_ge' :
+    (finrank ℚ K) ^ (finrank ℚ K * 2) / ((4 / π) ^ (nrComplexPlaces K * 2) *
```

Comment:
My instinct is to write `2 * x` instead of `x * 2` (for both the `finrank` and `nrComplexPlaces`), but this might just be based on the fact that humans would write $$2x$$ and never $$x2$$ in papers.

Reply:
Yes, it does look strange. I changed that.